tests/format/markdown/markdown/real-world-case.md

```diff
@@ -25,50 +25,70 @@ conforms to a consistent style. (See this [blog post](http://jlongster.com/A-Pre
 
 <!-- toc -->
 
-* [What does Prettier do?](#what-does-prettier-do)
-* [Why Prettier?](#why-prettier)
-  + [Building and enforcing a style guide](#building-and-enforcing-a-style-guide)
-  + [Helping Newcomers](#helping-newcomers)
-  + [Writing code](#writing-code)
-  + [Easy to adopt](#easy-to-adopt)
-  + [Clean up an existing codebase](#clean-up-an-existing-codebase)
-  + [Ride the hype train](#ride-the-hype-train)
-* [How does it compare to ESLint (or TSLint, stylelint...)?](#how-does-it-compare-to-eslint-or-tslint-stylelint)
-* [Usage](#usage)
-  + [CLI](#cli)
-  + [ESLint](#eslint)
-  + [Pre-commit Hook](#pre-commit-hook)
-  + [API](#api)
-  + [Excluding code from formatting](#excluding-code-from-formatting)
-* [Options](#options)
-  + [Print Width](#print-width)
-  + [Tab Width](#tab-width)
-  + [Tabs](#tabs)
-  + [Semicolons](#semicolons)
-  + [Quotes](#quotes)
-  + [Trailing Commas](#trailing-commas)
-  + [Bracket Spacing](#bracket-spacing)
-  + [JSX Brackets](#jsx-brackets)
-  + [Range](#range)
-  + [Parser](#parser)
-  + [Filepath](#filepath)
-* [Configuration File](#configuration-file)
-  + [Basic Configuration](#basic-configuration)
-  + [Configuration Overrides](#configuration-overrides)
-  + [Configuration Schema](#configuration-schema)
-* [Editor Integration](#editor-integration)
-  + [Atom](#atom)
-  + [Emacs](#emacs)
-  + [Vim](#vim)
-  + [Visual Studio Code](#visual-studio-code)
-  + [Visual Studio](#visual-studio)
-  + [Sublime Text](#sublime-text)
-  + [JetBrains WebStorm, PHPStorm, PyCharm...](#jetbrains-webstorm-phpstorm-pycharm)
-* [Language Support](#language-support)
-* [Related Projects](#related-projects)
-* [Technical Details](#technical-details)
-* [Badge](#badge)
-* [Contributing](#contributing)
+- [Prettier](#prettier)
```

Comment:
Can you revert change in this file?

Reply:
I now see this comment just above, but there doesn't seem to be a `yarn toc` script:

```
<!-- Do not edit TOC, regenerate with `yarn toc` -->
```

Reply:
This is only a test file, content don't need to be the same as the docs.